src/schema/Schema.ts

```diff
@@ -116,7 +116,7 @@ export class Schema {
       // missing foreign keys to the record first.
       if (key !== null) {
         // Using optional chaining in the event of an inverse polymorphic
-        // relation that won't have a defined relation.
+        // relation which won't have a defined relation.
         ;(parent.$fields()[key] as Relation)?.attach(parentRecord, record)
```

Comment:
Also, we can definitely remove the optional chain here as per [this comment](https://github.com/vuex-orm/vuex-orm-next/pull/107#discussion_r753888042).